src/model/cloud-runner/providers/k8s/kubernetes-task-runner.ts

```
@@ -22,94 +22,707 @@ class KubernetesTaskRunner {
+                  
+                  // Check node resources if pod is assigned
+                  if (podStatusDetails?.hostIP) {
+                    try {
+                      const nodes = await kubeClient.listNode();
+                      const hostIP = podStatusDetails.hostIP;
+                      const assignedNode = nodes.body.items.find((n: any) => 
+                        n.status?.addresses?.some((a: any) => a.address === hostIP)
+                      );
+                      if (assignedNode?.status && assignedNode.metadata?.name) {
+                        const allocatable = assignedNode.status.allocatable || {};
+                        const capacity = assignedNode.status.capacity || {};
```

ESLint

[error] 'capacity' is assigned a value but never used.

(no-unused-vars)